alicloud/service_alicloud_cas.go

```diff
@@ -0,0 +1,40 @@
+package alicloud
+
+import (
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/requests"
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/cas"
+	"github.com/terraform-providers/terraform-provider-alicloud/alicloud/connectivity"
+)
+
+type CasService struct {
+	client *connectivity.AliyunClient
+}
+
+func (s *CasService) DescribeCas(id string) (*cas.Certificate, error) {
+	var cert *cas.Certificate
+
+	args := cas.CreateDescribeUserCertificateListRequest()
+	args.ShowSize = requests.NewInteger(50)
```

Comment:
Please use `PageSizeLarge` instead of `50`.

In addition, I think there is a potential bug when there are more than 50 certificates.

Reply:
done